notification/shoutrrr_test.go

```diff
@@ -58,3 +60,23 @@ var _ = ginkgo.Describe("Notification properties", func() {
 		})
 	}
 })
+
+var _ = ginkgo.Describe("Shoutrrr", func() {
+	ginkgo.It("should template smtp", func() {
```

Comment:
Should we not be testing all the default templates ?

Reply:
Oh this was mainly to test the SMTP markdown -> html conversion.

I can add tests for all the templates.